packages/suite-desktop-core/e2e/playwright.config.ts

```diff
@@ -40,6 +40,12 @@ const config: PlaywrightTestConfig = {
         : [['list'], ['html', { open: 'never' }]],
     timeout: process.env.GITHUB_ACTION ? timeoutCIRun : timeoutLocalRun,
     outputDir: path.join(__dirname, 'test-results'),
+    snapshotPathTemplate: 'snapshots/{projectName}/{testFilePath}/{arg}{ext}',
```

Comment:
By default snapshots are stored in folder with test file. Which can lead to cluttered test folders. So this configuration puts snapshot to their own Folder.
On the other hand I am not sure how easy will be to find corresponding snapshot here. We will see and improve based on our experience